### PR TITLE
Patch1

### PR DESCRIPTION
**Pull request by clinkadink**

### CODE REVIEW

File: js/graphs.js [hunk 1]
```diff
@@ -434,39 +452,43 @@ function createGraph(graph, data) {
...
-	if (graph.blocksConfig && typeof (graph.blocksConfig['width']) !== 'undefined' && !graph.properties.popup) {
+	if (graph.blocksConfig && isDefined(graph.blocksConfig['width']) && !graph.properties.popup) {
 		width = graph.blocksConfig['width'];
 	}
 	var html = '';
 	html += title + '<div class="graphbuttons" >' + buttons + '</div>';
 	html += '</div>'
 
-	html += '<div class="graph swiper-no-swiping' + (graph.properties.popup ? ' popup graphheight' : '') + '" id="graph' + graph.properties.idx + '">';
+	html += '<div class="graph swiper-slide' + (graph.properties.popup ? ' popup graphheight' : '') + '" id="graph' + graph.properties.idx + '">';
```
Comment (lokonli) [inline js/graphs.js]: Adding class swiper-slide is not correct: It kills the graph height. The class swiper-no-swiping prevents to swipe to a next screen, if you swipe along the graph line on touch devices. I propose to undo the change of this line.

Comment (clinkadink) [reply js/graphs.js]: OK. Understood. The reason for updating it to '-slide' is because screens don't slide when you have multiple graphs filing the whole screen. The only way to get back to the previous screen (using touch or mouse) is to "try" and click the tiny margin in between the graphs.

Comment (lokonli) [reply js/graphs.js]: ok. The swiper-no-swiping is not really needed. Then let's just remove swiper-slide. ok?

Comment (clinkadink) [reply js/graphs.js]: Great compromise 👍 Thanks.
